notifier/plotting.go

```diff
@@ -86,22 +91,21 @@ func resolveMetricsWindow(logger moira.Logger, trigger moira.TriggerData, pkg No
 	// window is not wide: use shifted window to fetch extended historical data from graphite
 	if trigger.IsRemote {
 		if isWideWindow {
-			return alignToMinutes(fromTime.Unix()), toTime.Unix()
+			return fromTime.Unix(), toTime.Unix()
 		}
-		return alignToMinutes(toTime.Add(-defaultTimeRange + defaultTimeShift).Unix()), toTime.Add(defaultTimeShift).Unix()
+		return toTime.Add(-defaultTimeRange + defaultTimeShift).Unix(), toTime.Add(defaultTimeShift).Unix()
 	}
 	// resolve local trigger window
 	// window is realtime: use shifted window to fetch actual data from redis
 	// window is not realtime: force realtime window
 	if isRealTimeWindow {
-		return alignToMinutes(toTime.Add(-defaultTimeRange + defaultTimeShift).Unix()), toTime.Add(defaultTimeShift).Unix()
+		return toTime.Add(-defaultTimeRange + defaultTimeShift).Unix(), toTime.Add(defaultTimeShift).Unix()
 	}
-	return alignToMinutes(defaultFrom), defaultTo
+	return defaultFrom, defaultTo
 }
 
-func alignToMinutes(unixTime int64) int64 {
-	unixTime -= unixTime % 60
-	return unixTime
+func roundToRetention(unixTime int64) int64 {
+	return moira.RoundToNearestRetention(unixTime, defaultRetentionSeconds)
 }
```

Comment:
Not sure why we need to implement this simple function as we can use `moira.RoundToNearestRetention` with default retention everywhere above.

Reply:
I've introduced it for brevity since rounding happens like 10 times in `resolveMetricsWindow` and each time we round to same retention.